etrago/cluster/gas.py

```diff
@@ -732,7 +734,6 @@ def get_clustering_from_busmap(
     one_port_components = ["Generator", "Load", "Store"]
 
     for one_port in one_port_components:
-        one_port_components.remove(one_port)
         new_df, new_pnl = aggregateoneport(

```

Comment:
I'm wondering why these lines and the code in lines 748 - 759 is needed. To me it seems like they do the same. Can you help me?